### DISCUSSION TITLE
Fragment and Path used for resetPasswordForEmail

### DISCUSSION THREAD

**Discussion by scopethis** [supabase/supabase#4492] [Questions] [answered]
Is there a way for me to change the reset link path that is provided when a user requests a reset password link?
Currently it is redirected to ```/#access_token=xxxxx...``` 

I understand the security need for the fragment, but was wondering if it could be redirected to somewhere other than root?

Thanks.

**Comment by scopethis** [comment, marked as answer]
I guess one way to handle this is to detect the hash on the root and redirect from there with the hash intact...
```/#access_token=xxxxx...``` can be redirected via window location to ```/what/ever/we/want/#access_token=xxxxx...```

Sorry if this has been answered (or documented) elsewhere.
Thanks.

**Comment by J0** [comment]
Hey @scopethis,

Thanks for adding this question and answer! I'm sure it'll be helpful to other users.
